lib/holdings/item.rb

```diff
@@ -47,9 +47,11 @@ def library
       standard_or_zombie_library
     end
 
-    # @return [String]
-    def home_location
-      item_display[:home_location]
+    # This is generally the item's permanent location, but it can also be
+    # the temporary location if the location detail searchworksTreatTemporaryLocationAsPermanentLocation
+    # is set to true.
+    def effective_permanent_location_code
+      item_display[:effective_permanent_location_code] || item_display[:home_location]
```

Comment:
Is this conditional temporary, or something we'll want to keep for a long time?

Reply:
Just until we do a full reindex with the new name.